bin/edbgBmcWrapper.sh

```diff
@@ -4,8 +4,22 @@
 # help files will be searched under $EDBG_HOME/help/
 # ***************************************************************************
 export EDBG_HOME=/usr/
+PLAT_DTB_DIR=/etc/pdata
 
-export EDBG_DTB=/etc/pdata/rainier.dtb
+#Set default environment value to none
+EDBG_DTB=none
+
+for entry in "$PLAT_DTB_DIR"/*
+do
+  if [ -f "$entry" ];then
+     if [[ "$entry" == *".dtb"* ]]; then
+        EDBG_DTB=${entry}
+     else
+        echo "No DTB files found in the path"
```

Comment:
Is anything going to work if a dtb file isn't found?  If the answer is no, shouldn't this also be an exit of the script?

Reply:
yes. good point. done